atd-vzd/triggers/update_crash_city_id.sql

```diff
@@ -0,0 +1,29 @@
+-- This function updates the city id if a crashes position is moved into Austin
+CREATE OR REPLACE FUNCTION update_crash_city_id()
+    RETURNS trigger
+    LANGUAGE plpgsql
+AS $function$
+DECLARE 
+temprow RECORD;
+    BEGIN
+        -- loop through jurisdictions that contain the new position
+        -- if jurisdiction is in the list of valid jurisdictions then
+        -- update city id to be in Austin (22)
+        FOR temprow IN 
+        (
+            SELECT jurisdictions.id
+        FROM atd_txdot_crashes AS crashes
+            INNER JOIN atd_jurisdictions jurisdictions
+            ON (jurisdictions.geometry && NEW.position) AND ST_Contains(jurisdictions.geometry, NEW.position)
+        WHERE crashes.crash_id = NEW.crash_id
```

Comment:
👋 just eyeballing this sql, @roseeichelmann can you help me understand why it is necessary to join to `atd_txdot_crashes`? it looks to me like you only neext to select from `atd_jurisdictions` to get the info you need.

Reply:
no you are totally right, its not necessary! brain fart :p 

just pushed up changes